solc/CommandLineInterface.cpp

```diff
@@ -1379,6 +1391,24 @@ bool CommandLineInterface::processInput()
 		}
 	}
 
+	if (m_args.count(g_argModelCheckerEngine))
+	{
+		string engine = m_args[g_argModelCheckerEngine].as<string>();
+		if (engine == "none")
+			m_modelCheckerEngine = ModelCheckerEngine::None();
+		else if (engine == "bmc")
+			m_modelCheckerEngine = ModelCheckerEngine::BMC();
+		else if (engine == "chc")
+			m_modelCheckerEngine = ModelCheckerEngine::CHC();
+		else if (engine == "all")
+			m_modelCheckerEngine = ModelCheckerEngine::All();
```

Comment:
I think these strings should be globally defined in something like `g_modelCheckerEngineArgs`. That's how it's done for other options with arguments here.

Reply:
Not all, some are also done via strings. I can do that too. I did it this way to avoid more of these small definitions in this file.

Reply:
Ah, ok. These globals do add some clutter so if you want to avoid that it's fine with me. Personally I'd use them simply because with strings it's too easy to make a typo that won't be detected.

Reply:
Yea I agree with you that the globals are safer, will change.

Reply:
I see that you defined them but you're not using them here.

Reply:
True, forgot. Will update

Reply:
Now I'm actually only using them to display the options, since I'm using function `ModelChecker::fromString` for the conversion. Not sure it makes sense to keep the variables just for that @cameel

Reply:
I mean, if you already have the variables I'd use them. If you want to remove them that's also fine with me as I said above.